go.mod

```diff
@@ -17,3 +17,8 @@ require (
 	golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd // indirect
 	nhooyr.io/websocket v1.8.6
 )
+
+replace (
```

Comment:
This should be removed.

Reply:
Definitely - will remove it once those PRs merge.

Reply:
Removed (everything's been published now)